.travis.yml

```diff
@@ -3,7 +3,9 @@ cache:
   bundler: true
   directories:
     - node_modules
-rvm: 2.1.5
+rvm:
+  - 2.1.6
```

Comment:
@zendesk-shender I have a chef PR to move to 2.1.6 today, do you think we should skip that and go straight to 2.2.2?